### PR TITLE
fix bad reference to docker yaml

### PR DESCRIPTION
**Pull request by Mearman**

### CODE REVIEW

File: docs/website/kubernetes.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ Copy the [existing configuration](https://github.com/verdaccio/verdaccio/blob/ma
 and adapt it for your use case:
 
 ```bash
-wget https://raw.githubusercontent.com/verdaccio/verdaccio/master/conf/docker.yaml -O config.yaml
+wget https://raw.githubusercontent.com/verdaccio/verdaccio/master/packages/config/src/conf/docker.yaml -O config.yaml
```
Comment (juanpicado) [inline docs/website/kubernetes.md:75]: awesome, but you only need this file, the others are generated, could you undo the others?

Comment (juanpicado) [reply docs/website/kubernetes.md:75]: I forgot move this note to this repo https://github.com/verdaccio/website/blob/master/CONTRIBUTING.md#source-code-website

Comment (Mearman) [reply docs/website/kubernetes.md:75]: ah of course. I assumed the translations were manual rationalisations.

**Review by juanpicado — APPROVED** [review]
Thanks @Mearman  !!

File: docs/website/kubernetes.md (hunk 1 above)
Comment (juanpicado) [reply docs/website/kubernetes.md:75]: If you like you can update the contributing of this repo with the missing info from https://github.com/verdaccio/website/blob/master/CONTRIBUTING.md#source-code-website

The code was recently moved here

**Pull request merged**